packages/app-desktop/gui/NoteEditor/NoteEditor.tsx

```diff
@@ -656,7 +660,7 @@ const mapStateToProps = (state: AppState) => {
 		isSafeMode: state.settings.isSafeMode,
 		useCustomPdfViewer: false,
 		syncUserId: state.settings['sync.userId'],
-		shareCache: readFromSettings(state),
```

Comment:
`readFromSettings` returned a new object every time it was called, causing `NoteEditor` to rerender unnecessarily.